data/filters/wg21.py

```diff
@@ -361,10 +367,11 @@ def rm():  _diff('rmcolor', 'sout', 'del')
             pf.debug('mpark/wg21: stable name', target, 'not found')
             return link
 
-    note_cls = next(iter(cls for cls in elem.classes if cls in {'example', 'note', 'ednote'}), None)
+    note_cls = next(iter(cls for cls in elem.classes if cls in {'example', 'note', 'ednote', 'draftnote'}), None)
     if note_cls == 'example':  example()
     elif note_cls == 'note':   note()
-    elif note_cls == 'ednote': ednote(); return
+    elif note_cls == 'ednote': ednote()
```

Comment:
We should keep the `return` here. The intent here is to skip the `add`/`rm` handling that comes after, since we want `ednote` (as well as `draftnote`) to be blue even within `add` or `rm` blocks.

Reply:
Ah, this is me not understanding python :D